pkg/kubelet/cm/topologymanager/fake_topology_manager.go

```diff
@@ -37,7 +38,16 @@ func NewFakeManager() Manager {
 func NewFakeManagerWithHint(hint *TopologyHint) Manager {
 	klog.InfoS("NewFakeManagerWithHint")
 	return &fakeManager{
-		hint: hint,
+		hint:   hint,
+		policy: NewNonePolicy(),
+	}
+}
+
+// NewFakeManagerWithPolicy returns an instance of fake topology manager with specified policy
+func NewFakeManagerWithPolicy(policy Policy) Manager {
+	klog.InfoS("NewFakeManagerWithPolicy")
+	return &fakeManager{
+		policy: policy,
```

Comment:
I'm not a huge fan of introducing yet another `NewFakeManager()` function. What if we want a `FakeManager` with a hint and a policy set? We need to come up with something more generic.

Reply:
Ideally there should be a generic FakeManager(). I added since we already have one withAffinity, however underlying structure is one